guess-the-country/country-backend/country/api.py

```diff
@@ -56,7 +56,8 @@ def predict(file: UploadFile = File(...)):
 @api.post("/explain")
 async def explain_api(file: UploadFile = File(...)):
     model = load_model()
```

Comment:
Kannst du das bitte aus der Funktion ziehen? Sonst laden wir bei jeder Anfrage das Modell neu.

Für's erste kannst du einfach in predict.py das Modell laden und hierhin importieren (`from predict import model`), wir machen das später dann so, dass das Modell "lazily" geladen wird, d.h. es startet das Backend und lädt dann im Hintergrund das Modell.

Reply:
Danke für den Hinweis, hab ich gemacht aber in dem anderen PR "Delete CATEGORIES from predict file" (der unter diesem branch liegt: guess-the-country-bugfixes-backend-label-telaviv), da dieser sich dieser PR auf bugfixes bezieht. Der hier istnur für das Erstellen der Tests zuständig.